UI/js/getMenu.js

```diff
@@ -0,0 +1,43 @@
+window.addEventListener('load', (event) => {
+  event.preventDefault();
+  const meal = document.querySelector('div.meal-box-content');
+
+  const load = (foodMenu) => {
+    foodMenu.forEach((element) => {
+      const itemName = element.item_name;
+      const price = element.price;
+      const foodUrl = element.food_url;
+      const template = `
```

Comment:
I don’t think template is a good naming convention for this. Please refactor

Reply:
done